### ISSUE TITLE
OBS will "update settings" for any attached (USB) camera, throwing away your changes with every launch

### ISSUE THREAD

**Issue by lcalder** [obsproject/obs-studio#4244]
## Platform

Operating system and version: 10.0 Build 19042 (release: 2009; revision: 685; 64-bit)
OBS Studio version: OBS 25+ up to OBS 26.1.2-194-g744cb9752 (64-bit, windows)

## Expected Behavior

When one adds a USB video capture device and customizes the camera settings using "Configure Video" the settings should not disappear every time OBS is relaunched.

## Current Behavior

OBS will reset the camera settings to default EVERY TIME it launches, requiring a person to visit EVERY, SINGLE, camera and reestablish the correct settings (about 15 to 20 mouse clicks PER CAMERA depending on how many custom changes you made) on every relaunch.

## Steps to Reproduce

1. Add a usb camera
2. Adjust the camera settings by right-clicking the source and selecting "Properties" -> Configure Video
3. exit OBS
4. Relaunch OBS and all camera settings are back at their defaults... Voila!

## Additional information

Cameras that are "grabbed" by software drivers such as Xsplit VCam (provides virtual green screen) can configure camera settings that it uses without adjusting through OBS, and when exiting OBS, with the Xsplit VCam still running in the taskbar, you can open Xsplit after OBS has exited and still see your cameras set at your custom values. However, as soon as you launch OBS, the camera configuration will be reset to default, and will display those default results in both OBS and Xsplit VCAM.

**Comment by skeletonbow** [comment]
Is this a Logitech webcam by chance?  Sounds like the Logitech camera settings bugs in Logitech's drivers that everyone encounters.  It's possible that Xsplit has written a workaround for the Logitech bug.  OBS does not have such a workaround built into it currently but there are workarounds available for this Logitech bug from 3rd parties, namely StreamDeck plugin from BarRaider for camera settings, and CrashKoeck's STRM tool to name a few.  I haven't tested either personally yet but a number of others have and indicated that they work.

It would be great to see such a feature get directly integrated into OBS in the future as it is a major quality of life thing IMHO.  Logitech has dropped the bag on this one for almost 10 years.  I'm not aware of any other brand of camera that has this problem however so if you're using a non-Logitech camera it could be some other issue perhaps.

Just to be clear though, if this is a Logitech camera, it is not OBS throwing away your settings, it is Logitech's software doing it behind the scenes using archaic Windows APIs (and their newer GHUB software that claims to fix it does not do any better at all either).

**Comment by lcalder** [comment]
Wow. Thanks for the comprehensive response.
It just so happens that, ONE of my 3 cameras is a Logitech c930e, and I have searched in vain for logitech drivers / forums to find a solution.
On my other setup two of the three cameras are Logitech... which is obviously VERY unfortunate.

My noname cameras don't have this problem EXCEPT for one which was connected using Xsplit VCAM. Interestingly, just yesterday, Xsplit released a new version to me which said it "will no longer reset the camera settings when entering OBS..." so that means OBS wasn't the culprit after all, but Xsplit ON LAUNCH OF OBS, but NOT ON EXIT... So that is now saving my settings with the non-logitech camera.

When I had checked the OBS logs, it always says stuff like:
  [DShow Device: 'Side camera'] settings updated: 
  11:49:53.210: 	video device: USB Camera
  11:49:53.210: 	video path: \\?\usb#vid_0c45&pid_636b&mi_00#9&315a9f63&0&0000#{65e8773d-8f56-11d0-a3b9-00a0c9223196}\global
  11:49:53.210: 	resolution: 1920x1080
  11:49:53.210: 	flip: 0
  11:49:53.210: 	fps: 30.00 (inter
So I (wrongly) assumed obs must be doing something.
